FusionIIIT/applications/globals/models.py

```diff
@@ -41,7 +41,7 @@ class ExtraInfo(models.Model):
     sex = models.CharField(max_length=2, choices=Constants.SEX_CHOICES, default='M')
     age = models.IntegerField(default=18)
     address = models.TextField(max_length=1000, default="")
-    phone_no = models.IntegerField(default=15)
+    phone_no = models.BigIntegerField()
```

Comment:
What was the requirement?

Reply:
IntegerField have values from -2147483648 to 2147483647 so I can't store any mobile number like 9479309117